### ISSUE TITLE
Feature Request Review

### ISSUE THREAD

**Issue by mp1pro** [lilyinblue/bwa-u4-feature-request#1]
Great job referring to the WCAG 2.0 guidelines and taking less fortunate users into consideration. Allowing accessibility by using tabIndex is an intuitive and excellent feature. An example should have been given of how tabIndex would be used, however, I assumed it would be like `<div tabIndex="-1" />  `

Allowing users to search by pressing the enter key is a necessity these days as you have expounded on. we do not have to call the same` this.props.onSearch(this.state.term) ` that is already been called in the search function, it is better practice to call the search function instead. We should also use the attribute `onKeyPress={}` to call the enter key function. Here is the technical desing based on your description;

```
  handleKeyDown(event) {
    if(event.keyCode === 13) {
      this.search();
    }
}
```
and
`<input onKeyPress={this.handleKeyDown}/>`

The technical design for your feature request should contain code examples. Great feature request. Please do not mix reactjs with jQuery. You can dig into jQuerys library and get the vanilla code for implementing the enter key function and insert the same code into reactjs to get the same effect. We use reactjs so we do not directly manipulate html elements. We would rather render the changes instead.
